Autocoders/Python/schema/default/channel_schema.rng

```diff
@@ -23,14 +23,14 @@
 
 
             <oneOrMore>
-                <ref name = "channel_defenition"></ref>
+                <ref name = "channel_definition"></ref>
```

Comment:
I'm not sure about this file format, it's possible this is an API change

Reply:
This is the correct change, but I will loop back and look at this after I think of the impact of this change.

Reply:
This should be ok.